### DISCUSSION TITLE
TumblThree doesn't run on my other PC

### DISCUSSION THREAD

**Discussion by tacphotog** [TumblThreeApp/TumblThree#399] [Q&A (Question & Answer)] [answered]
I downloaded TumblThree and executed the file.  After the program extracted I double clicked on the .exe file and I get a box that pops up and says that "You need to install an app for this task.  Would you like to search for one in the Store?"  If you say no nothing happens.  If you say yes it opens the MS store app and it says it can't find an app to open the task.  It doesn't say what app it needs.  I'm running Windows 11 on a new PC.  TumblThree ran fine on my old PC.  This is the only error I see in the log.

20221112 10:51:35.242	Err 	LogService.NetVersionSupport: System.ComponentModel.Win32Exception (0x80004005): The system cannot find the file specified
   at System.Diagnostics.Process.StartWithCreateProcess(ProcessStartInfo startInfo)
   at TumblThree.Applications.Services.LogService.get_NetVersionSupport() in C:\projects\Tumblthree\src\TumblThree\TumblThree.Applications\Services\LogService.cs:line 383

**Comment by thomas694** [comment]
The needed .NET framework is shipped with Windows 10 / 11 and is expected to work. Actually TumblThree starts and can write the log file, so that seems to be ok. Can you successfully start the image viewer which is included in the installation folder? What happens after you close the MS store app? TumblThree is usable or not? You can also have a look in the Windows event viewer after the message appeared and the app store opened.

The quoted error which shouldn't show a message and just be logged, is expected and means that this PC has neither .NET 5 nor 6 installed. If you want you can already install the .NET 6 Runtime ([x64](https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/runtime-desktop-6.0.11-windows-x64-installer) / [x86](https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/runtime-desktop-6.0.11-windows-x86-installer)).

**Comment by tacphotog** [comment]
I totally uninstalled the program, installed .net 6 and rebooted.  I uninstalled TumblThree and then re-downloaded and installed it.  I does the same thing, says I need to download something.  It did not write a log file this time.  Nothing happens when I close the MS store app.  The same problem happens when I try to run the image viewer.  TumblThree is not usable at all.  I didn't see anything in event viewer.

**Comment by thomas694** [comment]
It looks like the framework installation on your PC has problems running any WPF application.
I just tested it on a fresh Windows 11 and TumblThree runs without a problem, it's not necessary to download or install anything.
You can try to repair your .NET Framework installation with "Repair" under "Programs and Features" or maybe this tool ([Microsoft .NET Framework Repair Tool](https://support.microsoft.com/en-us/topic/microsoft-net-framework-repair-tool-is-available-942a01e3-5b8b-7abb-c166-c34a2f4b612a)).

**Comment by tacphotog** [comment]
I tried it and it didn't work.

**Comment by thomas694** [comment, marked as answer]
Do you have the same problem with other (downloaded) applications too?
You can also try the following, one step at a time to see which one really helped.
Before extracting the TumblThree zip file, make sure that it's "unblocked" (right-click on zip file -> Properties). Should be automatically, but check the same on the extracted TumblThree.exe.
Temporarily disable the option "Check apps and files" under "App & browser control" in settings.

**Comment by tacphotog** [comment]
That did it.  The zip file was blocked.  I learned something new.  Thanks so much.  I love the program and it's good to have it back working.
